scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
Yes, it's an extension, but it's one that every sh I have access to (dash, posh, mksh, ksh, FreeBSD/OpenBSD/NetBSD sh) implements.  We use that in most of our scripts.